main.tex

```diff
@@ -1303,9 +1316,7 @@
 
 \item  keep their Lot secure;
 
-\item  ensure any garden area associated with their Lot for their exclusive use is maintained (i.e. lawn area mowed, plants maintained, shrubs/trees cut back etc.); and
```

Comment:
As mentioned before, the designs for this building included plants which are intended to grow over edges and up walls.

"Maintaining" a plant is completely subjective.